plugins/pencilblue/controllers/index.js

```diff
@@ -275,10 +275,9 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
     ats.registerLocal('author_position', content.author_position ? content.author_position : '');
     ats.registerLocal('media_body_style', content.media_body_style ? content.media_body_style : '');
     ats.registerLocal('comments', function(flag, cb) {
-       if (isPage || !contentSettings.allow_comments) {
-           cb(null, '');
-           return;
-       }
+        if (isPage || !contentSettings.allow_comments || !content.allow_comments) {
```

Comment:
If someone upgrades and they have articles saved in the system from before this functionality, they will never show comments. I suggest changing this to:
```JavaScript
if (isPage || !contentSettings.allow_comments || content.allow_comments === false) {
```